### DISCUSSION TITLE
Directus Hosting Recommendations

### DISCUSSION THREAD

**Discussion by MentalGear** [directus/directus#16565] [Archive]
### Use Case
Support different tenants in a "Organisation > Teams > Project" structure

### Method
It seems the only current way to do this is to have different `directus` instances (one for each 'org') which the `directus structure` synced from a "master structure" using cli's `migrate`.

### Options
I'd be really like to use the official Directus Cloud for hosting and support the general project, but since it doesn't allow underlying db / server access to use "migrate", I'd like to ask what other Directus Services you can recommend?

**Comment by wfriedman89** [comment]
@MentalGear Directus Enterprise Cloud allows for direct DB access and use of the schema migrate tool to migrate db schema from one instance to another.  If you are interested in Directus Enterprise Cloud reach out to us here:  https://directus.io/contact/
